### ISSUE TITLE
Remove inappropriate Token struct

### ISSUE THREAD

**Issue by kishansagathiya** [fabric8-services/fabric8-jenkins-proxy#190]
From what this struct contains it doesn't seem like a token to me https://github.com/fabric8-services/fabric8-jenkins-proxy/blob/master/internal/util/osio_token.go#L39

And token is exactly same as User. Probably we should get rid of Token and use User instead.
User struct is here https://github.com/fabric8-services/fabric8-jenkins-proxy/blob/master/internal/util/osio_token.go#L16

Code snippet which I am talking about
```

type User struct {
	Data Data `json:"data"`
}

type Data struct {
	Attributes Attributes `json:"attributes"`
}

type Attributes struct {
	Username string `json:"username"`
	Email    string `json:"email"`
	FullName string `json:"fullName"`
}

type Result struct {
	Tokens []TokenResult `json:"tokens"`
}

type TokenResult struct {
	UUID  string `json:"uuid"`
	Token string `json:"token"`
}

type Token struct {
	Data Data `json:"data"`
}
```

**Comment by hferentschik** [comment]
Not sure tbh. It might be redundant, it might not be. Needs to be investigated. @vpavlin any input on this?

**Comment by kishansagathiya** [comment]
Removing Token, TokenResult, and Result doesn't break anything

**Comment by hferentschik** [comment]
Merged via pull request #191
